packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -33,6 +32,8 @@ export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
   open: boolean;
   placement?: PopperPlacementType;
   popperOptions?: object;
+  popperRef?: React.RefObject<PopperJs>;
```

Comment:
```suggestion
  popperRef?: React.Ref<PopperJs>;
```

which includes callback refs. Callback refs are supported by `useForkRef`

Reply:
Why was this discarded?